frontend/src/js/components/releases/dialogs/AddArtifact.tsx

```diff
@@ -67,29 +90,48 @@ const shortenFileName = name => {
   return name;
 };
 
-export const ArtifactUpload = ({ setSnackbar, updateCreation }) => {
+const singleFileLimit = 536870912; //512MiB
```

Comment:
maybe it's better to move those consts in const file and write them as `512 * (1024 ** 2)` and `10 * (1024 ** 3)`?

Reply:
Since this is very isolated I wouldn't want to move it anywhere else until it gets reused in a different context.